OpenJDK_Playlist/playlist.xml

```diff
@@ -352,23 +335,6 @@
 		</tags>
 	</test>
 	<test>
-		<testCaseName>jdk_sound</testCaseName>
-		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(JTREG_DIR)$(D)lib$(D)jtreg.jar$(Q) \
-	$(JTREG_BASIC_OPTIONS)  -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS)$(Q) \
-	-w $(Q)$(TEST_RESROOT)$(D)work$(Q) \
-	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
-	-jdk:$(Q)$(JDK_HOME)$(Q) \
-	-exclude:$(Q)$(JTREG_TEST_DIR)$(D)ProblemList.txt$(Q) \
-	$(Q)$(JTREG_TEST_DIR):jdk_sound$(Q); \
-	$(TEST_STATUS)</command>
-		<subsets>
-			<subset>SE80</subset>
-		</subsets>
-		<tags>
-			<tag>extended</tag>
-		</tags>
-	</test>
-	<test>
 		<testCaseName>jdk_swing</testCaseName>
```

Comment:
There are also many x11 server failures in the jdk_swing group, if you are temporarily excluding tests have known failures of great numbers you may want to include it.